src/components/aktivitetskrav/vurdering/VurderAktivitetskravButtons.tsx

```diff
@@ -43,9 +52,15 @@ export const VurderAktivitetskravButtons = ({
           {texts.forhandsvarsel}
         </Button>
       )}
-      <Button variant="secondary" onClick={() => onButtonClick("IKKE_OPPFYLT")}>
-        {texts.ikkeOppfylt}
-      </Button>
+      {hasUbehandletVurderStansOppgave &&
```

Comment:
Knappen må vel være tilgjengelig som før for de som ikke har toggle enabled :monocle_face:

Reply:
Stemme! Bra sett 🙌